src/app/myday/myday.page.ts

```diff
@@ -53,6 +64,10 @@ export class MydayPage implements OnInit, OnDestroy {
     this.alertReminderService.presentAlertPrompt('Add');
   }
 
+  segmentChanged(ev: any) {
```

Comment:
i see what you have tried to do here. but if you can simply use template variable it will remove the need of this function as well as maintaining this segment variable.